apps/docs/src/components/ex/List/style.css

```
@@ -1,34 +1,34 @@
+    --li-ps: 1.875em;
+    /* --icon-clip: circle(0.225em at 50%); */
   }
 
   /* ボーダースタイルをつけれるように、li 側の ps で余白調整をする */
   .b--list > li {
     position: relative;
-    padding-inline-start: var(--_li-ps);
+    padding-inline-start: var(--li-ps);
   }
 
   /* テキスト（半角スペース）を入れて、疑似要素の高さをリストのテキストに合わせる。（line-heightも含めて高さを一致させる）*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)